Dockerfile.updater-core

```
@@ -10,12 +10,25 @@ ENV DEBIAN_FRONTEND="noninteractive" \
   LC_ALL="en_US.UTF-8" \
   LANG="en_US.UTF-8"
 
+# Use a custom sources.list to use Azure's mirrors which are faster
+ADD ubuntu.sources /etc/apt/sources.list.d/ubuntu.sources
+
 RUN apt-get update \
   && apt-get upgrade -y \
   && apt-get install -y --no-install-recommends \
+    # needed to run add-apt-repository
+    software-properties-common \
+    # Used temporarily to download git lfs install script as well as dev dependencies for CI
+    curl \
+    # Add git core ppa to get a more recent git version than the one provided by ubuntu 24.04 which is 2.43.0(released in November 20, 2023)
```

nit: this comment will be out of date if/when we update Ubuntu or Ubuntu updates git

```suggestion
    # Add git core ppa to get a more recent git version than the one provided by ubuntu
```